### PR TITLE
1.2.0 Feature: REST: Channel Encryption (set channel options)

### PR DESCRIPTION
**Pull request by QuintinWillison**
One of a family of pull requests addressing https://github.com/ably/features/issues/3.

I could see no evidence that either the Java / Android or Ruby SDKs had methods to allow you to set options for REST channel instances.

### CODE REVIEW

File: sdk.yaml [hunk 1]
```diff
@@ -374,6 +374,11 @@ REST:
       .specification: RSA8e
   Channel:
     .specification: [RSN1, RTC3]
+    Encryption:
+      .specification: [RSL7, TB2b, RSN3c]
+      .synopsis: |
+        The option to configure a `cipher`, specifying encryption algorithm and attributes.
```
Comment (lmars) [inline sdk.yaml:380]: Is it appropriate to documented exactly what kind of encryption we're talking about here (i.e. AES-CBC with either 128 or 256 bit keys)?

Comment (QuintinWillison) [reply sdk.yaml:380]: I think that makes sense. I'll borrow words from here: https://ably.com/docs/api/realtime-sdk/encryption#cipher-params

Comment (QuintinWillison) [reply sdk.yaml:380]: https://github.com/ably/features/pull/29/commits/29353be618e2905f77926d7927e5f176d1a91244

Comment (QuintinWillison) [reply sdk.yaml:380]: Resolving this conversation on @lmars' behalf so that I can get this pull request landed.

**Comment by QuintinWillison** [comment]
Having addressed @lmars' comment, I'm going to merge this pull request without waiting for approval. It has been open for a while and there is a stack of pull requests ahead of it. We are likely to have another look at the entire work at the end, to double check that it congeals holistically, so the risk of merge without approval is low.

**Pull request merged**